builds/wasm/src/vade/vade-api-sidetree.ts

```diff
@@ -0,0 +1,157 @@
+import type { DIDDocument } from './interfaces';
+import {
+  DidCreateOptions,
+  DidCreatePayload,
+  DidCreateResponse,
+  DidDeactivatePayload,
+  DidRecoverPayload,
+  DidUpdateOptions,
+  DidUpdatePayload,
+  JsonWebKeyWithNonce,
+  Patch,
+  PublicKeyModel,
+  Service,
+} from './typings/vade-sidetree';
+
+import { VadeApiShared } from './vade-api-shared';
+
+export enum UpdateType {
+  Update = 'update',
+  Recovery = 'recovery',
+  Deactivate = 'deactivate',
+}
+
+export enum PublicKeyPurpose {
+  Authentication = 'authentication',
+  AssertionMethod = 'assertionMethod',
+  CapabilityInvocation = 'capabilityInvocation',
+  CapabilityDelegation = 'capabilityDelegation',
+  KeyAgreement = 'keyAgreement',
+}
+
+const TYPE_SIDETREE = 'sidetree';
+class VadeApiSidetree extends VadeApiShared {
```

Comment:
Could you add function docs? ^^

Also `createDid`'s function doc is rather brief regarding the arguments, and `getDid` documents more arguments than it has. ^^